desktop-respons.css

```diff
@@ -0,0 +1,241 @@
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .header-at-top {
+    background-image: url(./img/Header\ bg.svg);
+    background-size: cover;
+  }
+
+  .menu-phone {
+    display: none;
+  }
+
+  .header-containers {
+    background-color: #fff;
+  }
+
+  .logo {
+    padding-left: 143px;
+    color: #6070ff;
+  }
+
+  .navbar-items {
+    padding-right: 142px;
+  }
+
+  .navbar-items a {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #6070ff;
+    text-decoration: none;
+    padding: 0 12px;
+  }
+
+  .primary-text,
+  .social,
+  .social-media-icons {
+    padding-left: 260px;
+  }
+
+  .text-about {
+    padding: 0 260px;
+  }
+
+  .work-container {
+    display: grid;
+    column-gap: 0;
+    row-gap: 88px;
+  }
+
+  .box {
+    display: flex;
+    flex-direction: row;
+    padding: 24px;
+    width: 60%;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 20px;
+    margin-left: auto;
+    margin-right: auto;
+    height: fit-content;
+  }
+
+  .more-details {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    width: 544px;
+    height: fit-content;
+    align-items: flex-start;
+  }
+
+  .snapshot {
+    width: 554px;
+    border-radius: 1px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    padding: 114px 96px;
+    width: 100%;
+    background-color: #fff;
+    border-top-right-radius: 150px;
+  }
+
+  .social-media-icons2 {
+    padding-left: 0;
+  }
+
+  .social-media-icons li {
+    display: inline;
+  }
+
+  .intro {
+    width: 40%;
+  }
+
+  .language-program {
+    width: 60%;
+    padding: 24px;
+    margin: 0;
+    align-items: flex-start;
+    height: 409px;
+  }
+
+  .language-program li {
+    padding: 0;
+    width: 100%;
+  }
+
+  .language-title {
+    padding-right: 31%;
+  }
+
+  .language-list {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .language-list li {
+    flex-direction: column;
+    height: fit-content;
+  }
+
+  .frameworks,
+  .skills {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding: 18px 0;
+    width: 100%;
+  }
+
+  .frameworks-title,
+  .skills-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+  }
+
+  /* contact section */
+
+  .contact {
+    padding: 122px 200px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
```

Comment:
- Hey, let's kindly also add the missing desktop background image of this section to fix the issue with no image is this part of the page. You can refer to the Figma template to add the missing image.